packages/client/src/components/Geolocation/index.tsx

```diff
@@ -0,0 +1,59 @@
+import {useEffect} from 'react';
+import {useAppDispatch} from 'client/src/hooks/redux';
+import {geolocationSelectors} from 'client/src/stores/reducers/geolocation/GeolocationReducers';
+import {useAppSelector} from 'client/src/hooks/redux';
+import {PirateFlag} from '../images/PirateFlag';
+import './Geolocation.scss';
+
+export const Geolocation = () => {
+	const dispatch = useAppDispatch();
+	const geolocation = useAppSelector(geolocationSelectors.geolocation);
+
+	useEffect(() => {
+		if (navigator.geolocation) {
+			navigator.geolocation.getCurrentPosition(
+				position => {
+					const {longitude} = position.coords;
+					fetch(
+						// eslint-disable-next-line max-len
+						`https://api.bigdatacloud.net/data/reverse-geocode-client?latitude=&longitude=${longitude}&localityLanguage=ru`,
+					)
+						.then(response => response.json())
+						.then(data => {
+							const action = {
+								type: 'actionGeolocation',
+								countryName: data.countryName,
+								countryCode: data.countryCode,
+							};
+							dispatch(action);
+						})
+						.catch(error => console.log(error));
```

Comment:
Если это ошибка, выводи ее как ошибку
```suggestion
						.catch(error => console.error(error));
```